README.asciidoc

```diff
@@ -0,0 +1,132 @@
+== Mainboard
```

Comment:
README and all other files should be in `electronics` directory.

`picr22-team-snax-mainboard` should be renamed to `electronics`.

These files should be removed:

* BOM.png
* picr22-team-snax-mainboard_v3011.zip
* picr22-team-snax-mainboard/Mainboard_documentation_-_team_sn2x.pdf
* picr22-team-snax-mainboard/Project Outputs for picr22-team-snax-mainboard.zip

Also History, Project Logs, __Previews directories should be removed.

Reply:
renamed, deleted and moved all the files to the `electronics` directory